templates/conjur.conf.j2

```diff
@@ -1,5 +1,5 @@
 account: {{conjur_account}}
 appliance_url: {{conjur_appliance_url}}
-cert_file: /etc/conjur.pem
+cert_file: {{conjur_ssl_certificate_path}}
```

Comment:
It feels like cert path should be optionally inserted into the file.  Do you think it makes sense to leaving it blank when there is no cert?

Reply:
Yes, all the solutions have the same capabilities:
1. HTTP
2.HTTPS : tested only with self signed cert (you can see the command in the nginx Docker file)
CA-cert: I believe its works also but not tested yet (low priority )

What do you mean by normal signed?

Reply:
I mean a standard issue cert signed by a provider like Komodo, Digisign, or GoDaddy.